app/models/user.rb

```diff
@@ -9,7 +9,7 @@ class User < ActiveRecord::Base
 
   include ActionView::Helpers::SanitizeHelper
 
-  validates :name, length: { minimum: 4, maximum: 70 }, format: {
+  validates :name, length: { minimum: 1, maximum: 70 }, format: {
```

Comment:
Should have done this in another PR, but OK since it's so simple. Please mention both issues in your PR description.

Reply:
Yeah, that was a mistake with git branching but for that same reason of simplicity and the fact that we all spoke about it together, I didn't fix it

Reply:
Will change description

Reply:
Description updated 4 hours ago